src/dawn/Optimizer/PassDataLocalityMetric.cpp

```diff
@@ -125,13 +128,23 @@ class ReadWriteCounter : public ASTVisitorForwarding {
     if(cache.getCacheType() == Cache::K) {
       if(cache.getCacheIOPolicy() == Cache::fill ||
          cache.getCacheIOPolicy() == Cache::fill_and_flush) {
-        std::cout << "   Fill: " << getNameFromAccessID(AccessID) << std::endl;
+//        std::cout << "   Fill: " << getNameFromAccessID(AccessID) << std::endl;
```

Comment:
this is dead code, can we remove it?

Reply:
Well, the whole pass could be refactored. These were just cout's in the middle of the computation. We can go one of two ways: Either we let the pass return a struct that holds all the info that is cout'ed and store it somewere (probably in the same place where the hardware-config singleton should be) or we make it s.t the pass only keeps the data locally and add a proper if verbose-stmt to it.

For now I did not want to kill all the output that the pass had for good so I left in in this stage until the decision is made